classes/Hook.php

```diff
@@ -683,6 +683,8 @@ public static function exec($hook_name, $hook_args = array(), $id_module = null,
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$hook_name, $hook_args);
                 } elseif ($hook_retro_callable) {
                     $display = Hook::coreCallHook($moduleInstance, 'hook'.$retro_hook_name, $hook_args);
+                } else {
+                    continue;
```

Comment:
Hi @firstred,

We only have to see this point before I can merge the pull request.
I hope to read you soon :)